.clusterfuzzlite/build.sh

```
@@ -0,0 +1,39 @@
+python3 generate_proto.py
+
+# Install the SDK and all runtime dependencies (uses current CFLAGS/CXXFLAGS so
+# any C extensions such as grpcio and cryptography are built with the right flags).
+pip3 install .
+
+# Install Atheris (the Python fuzzing engine) and PyInstaller (used to produce
+# self-contained fuzzer executables that ClusterFuzzLite can run reliably).
+pip3 install atheris pyinstaller
+
+# Build every *_fuzzer.py target found in the .clusterfuzzlite directory.
+for fuzzer in $(find "$SRC/hiero-sdk-python/.clusterfuzzlite" -name '*_fuzzer.py'); do
```

Shellcheck (0.11.0)

[warning] For loops over find output are fragile. Use find -exec or a while read loop.

(SC2044)